Rent-A-Rig/WebContent/html/index-footer.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="../css/mainStyles.css">
```

Comment:
the index-footer should link to css/mainStyles instead of ../css/mainStyles

Reply:
Just change it ! Thanks!